features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,20 @@
 Then("I should see {string}") do |content|
   expect(page).to have_content content
 end
+
+Then("{string} is present") do |selector|
+  expect(page).to have_selector :css, "#navigationBar"
+  expect(page).to have_selector :css, "#footer"
+end
+
+Then("I would like to see a {string} link") do |link|
+  find_link link
+end
+
+Then("I would like to click on {string} link") do |link|
+  click_link_or_button link
```

Comment:
The title sounds like an assertion but you give a command to Capybarawhat to click on.

Reply:
Changed to `Then I click on "Home" link`

Reply:
I really don't understand... `Then` steps are usually assertion steps. What is going on here?